src/sir.c

```diff
@@ -252,3 +252,5 @@ uint32_t sir_getversionhex(void) {
 bool sir_isprerelease(void) {
     return (0 == SIR_VERSION_IS_RELEASE);
 }
+
+void *ignore_memory_leak;
```

Comment:
???

Reply:
Definitely hate more than any similar addition so far lol

Reply:
It's what Qt does :(